docs/pico_sdk.md

```diff
@@ -50,7 +53,7 @@ git submodules. for more detail, see the below instructions to incorporate RF24
 In order to use the RF24 libraries in your RP2040 based project:
 
 1. Make a copy of the RF24 library (and optionally RF24Network and RF24Mesh
-   libraries) in a "lib" directory located in your project's root directory.
+   libraries) in a `lib` directory located in your project's root directory.
```

Comment:
I actually prefer this be in quotes because inline code blocks show up different in doxygen (barely noticable), and the word lib is not actual code.

Reply:
Sure, no problem. Changed.